content/layout/flexbox.md

```diff
@@ -1 +1,347 @@
+
+1. `main axis` The main axis of a flex container is the primary axis along which flex items are laid out. Beware, it is not necessarily horizontal; it depends on the flex-direction property (see below).
+2. `main-start | main-end` The flex items are placed within the container starting from main-start and going to main-end.
+3. `main size` A flex item's width or height, whichever is in the main dimension, is the item's main size. The flex item's main size property is either the ‘width’ or ‘height’ property, whichever is in the main dimension.
+4. `cross axis` The axis perpendicular to the main axis is called the cross axis. Its direction depends on the main axis direction.
+5. `cross-start | cross-end` Flex lines are filled with items and placed into the container starting on the cross-start side of the flex container and going toward the cross-end side.
+6. `cross size` The width or height of a flex item, whichever is in the cross dimension, is the item's cross size. The cross size property is whichever of ‘width’ or ‘height’ that is in the cross dimension.
+
+
+
+## Flexbox properties
+
+There are several new properties introduced by flexbox. Some of them affect the parent element and some of them its children.
+
+**Parent flexbox properties**
+
+1. [display](#display)
+2. [flex-direction](#flex-direction)
+3. [flex-wrap](#flex-wrap)
+4. [flex-flow](#flex-flow)
+5. [justify-content](#justify-content)
+6. [align-items](#align-items)
+7. [align-content](#align-content)
+
+**Children flexbox properties**
+
+1. [order](#order)
+2. [flex-grow](#flex-grow)
+3. [flex-shrink](#flex-shrink)
+4. [flex-basis](#flex-basis)
+5. [flex](#flex)
+6. [align-flex](#align-flex)
+
+### Parent flexbox properties
+
+#### display
+
+It enables flexbox in the element. 
+
+```css
+/* possible values */
+
+.flex {
+    display: flex;
+}
+
+.flex {
+    display: inline-flex;
+}
+```
+
+__Demo__: http://codepen.io/asainz/pen/zGPKOv  
+__Gist__: https://gist.github.com/asainz/79f3771b97ec05828686  
+
+
+#### flex-direction
+
+This establishes the main-axis. This means, which direction the children items will follow (horizontal or vertical).
+
+```css
+.flex{
+    flex-direction: row; /* default */
+}
+
+.flex{
+    flex-direction: row-reverse;
+}
+
+.flex{
+    flex-direction: column;
+}
+
+.flex{
+    flex-direction: column-reverse;
+}
+```
+
+__Demo__: http://codepen.io/asainz/pen/BNmLQG  
+__Gist__: https://gist.github.com/asainz/6be47ef9f134978275a9  
+
+#### flex-wrap
+
+Flex items will all try to fit onto one line. You can change that and allow the items to wrap as needed with this property.
+
+```css
+.flex{
+    flex-wrap:  nowrap; /* default */
+}
+
+.flex{
+    flex-wrap: wrap;
+}
+
+.flex{
+    flex-wrap: wrap-reverse;
+}
+```
+
+__Demo__: http://codepen.io/asainz/pen/doZpWZ  
+__Gist__: https://gist.github.com/asainz/bbb5e721f37e84c19965  
+
+#### flex-flow
```

Comment:
I'd put this right below `flex-direction` and `flex-wrap`, the reader will find it easy to recall.